### ISSUE TITLE
Upgrade flowplayer to latest (6.0.5) version

### ISSUE THREAD

**Issue by nadavkav** [h5p/flowplayer#1]
Please see if you can upgrade to the new and latest version of FlowPlayer.
https://www.flowplayer.org/blog/flowplayer-6.0/

As we are trying to move forward and use MPEG-DASH to stream our videos with H5P interactions.
https://www.flowplayer.org/docs/plugins.html#dashjs

**Comment by nadavkav** [comment]
Suggested PR #2 
(Attaching a working test of H5P Interactive video which is using the above PR)
[interactive-video-24-8068.h5p.zip](https://github.com/h5p/flowplayer/files/344589/interactive-video-24-8068.h5p.zip)

**Comment by nadavkav** [comment]
Maybe we should split it. An updated flowplayer and a flowplayer + dashjs ?

**Comment by falcon-git** [comment]
Thanks! Reviewing this has been scheduled for next release. (Not the one we're working on now, but the one after this one. probably starts working on it in August)

You have noticed that Flowplayer is only used as a fallback for browsers not supporting HTML5 video.

**Comment by nadavkav** [comment]
Ok.
I do not use it as a fallback, but as a base for MPEG-DASH support.

I am also on progress of creating a new sub video plugin that implements videojs with some of its sub plugins, as alternative to this flowplayer plugin. (so you know)

**Comment by falcon-git** [comment]
Ok, I think we would prefer to implement dash support in H5Ps own players instead of using these huge third party video player libraries. Do you see other benefits with going for Flowplayer og videojs?

**Comment by nadavkav** [comment]
No benefits in flowplayer over Videojs. At the time (2-3 years ago), it was a good fallback for anyone needed flash. but not anymore.

I was actually surprised they have an updated version (6.0.5), as I though the project was not [getting any progress](https://github.com/flowplayer/flowplayer/graphs/contributors) for a long time, and was also surprised they had DASH-MPEG plugin (which is a wrapper to official dash.js).
Seems like they are "back in business", but videojs have [10 times more contributors](https://github.com/videojs/video.js/graphs/contributors), features and [plugins](https://github.com/videojs/video.js/wiki/Plugins)

I do not think you should bother implementing MPEG-DASH at core H5P on your own, better have it as a plugin under H5P.Video . I have it as a proof of concept, added to the current flowplayer, only because you are already using it in the H5P project. now that I have basic understanding of your video player I will definitely use videojs.

btw, see:
http://html5video.org/wiki/HTML5_Video_Player_Comparison
http://videosws.praegnanz.de/

**Comment by nadavkav** [comment]
One more thing about the #2 , in case you at least consider updating the flowplayer code from v3 to v6 ...
The API changed. so H5P.Video (https://github.com/h5p/h5p-video/blob/master/scripts/flash.js) needs some work. I made some updates that I can PR into H5P.Video . so just say so, when you have time to review it (above PR)

**Comment by falcon-git** [comment]
Ok, I think we might want to use flowplayer v3 since we're only using it as a fallback for ppl with very old browsers without HTML5 support. We might even remove Flowplayer alltogether now.

What do you think are the benefits of using Flowplayer/VideoJS instead of just adding dash.js to our own players? Is it worth the extra weight?

**Comment by icc** [comment]
I like the idea of dropping the old Flowplayer. Browser video support has been around for quite a while now, and it also looks like the major browsers will start dropping Flash support very soon.

**Comment by nadavkav** [comment]
Indeed, minimized VideoJS takes 58K, were minimized Flowplayer takes only 13K. but looking at the benefits, it seems like a good choice:
http://blog.videojs.com/Video-js-5-The-Only-Thing-That%E2%80%99s-Changed-Is-Everything-except-for-like-3-things-that-didn-t-including-the-name/
Plugins: https://github.com/videojs/video.js/wiki/Plugins
Some "popularity contest": https://js.libhunt.com/project/video-js/vs/flowplayer?rel=cmp-lib

**Comment by icc** [comment]
It does appear to be more popular and have a bigger growing community.
